### PR TITLE
[14.0] [MIG] delivery_free_fee_removal

### PR DESCRIPTION
**Pull request by sonhd91**

### CODE REVIEW

File: delivery_free_fee_removal/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Delivery Free Fee Removal",
     "summary": "Hide free fee lines on sales orders",
-    "version": "13.0.1.0.2",
+    "version": "14.0.0.0.0",
```
Comment (rousseldenis) [inline delivery_free_fee_removal/__manifest__.py]: Please change to 14.0.1.0.0

File: delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py [hunk 2]
```diff
@@ -52,8 +52,8 @@ def test_delivery_free_fee_removal_with_fee(self):
             [
                 {
                     "is_free_delivery": False,
-                    "qty_to_invoice": 1,
-                    "invoice_status": "to invoice",
+                    "qty_to_invoice": 0.0,
```
Comment (rousseldenis) [inline delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py]: Why these changes in tests ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py (hunk 2 above)
Comment (sonhd91) [reply delivery_free_fee_removal/tests/test_delivery_free_fee_removal.py]: Odoo14 prevent trigger sale_order.product_id.invoice_status after confirm, Sale order not yet delivered so nothing to invoice, I add more test case for this module in PR

File: delivery_free_fee_removal/__manifest__.py (hunk 1 above)
Comment (sonhd91) [reply delivery_free_fee_removal/__manifest__.py]: Code refactored

**Review by sebalix — APPROVED** [review]
LG

**Review by mmequignon — APPROVED** [review]

**Comment by simahawk** [comment]
@rousseldenis good for you?
